spec/graphql/static_validation/rules/argument_literals_are_compatible_spec.rb

```diff
@@ -333,6 +333,35 @@
     end
   end
 
+  describe "error references argument" do
+    let(:validator) { GraphQL::StaticValidation::Validator.new(schema: schema) }
+    let(:query) { GraphQL::Query.new(schema, query_string) }
+    let(:errors) { validator.validate(query)[:errors] }
+    let(:query_string) {"
+      query {
+        cheese(id: true) { source }
+        milk(id: 1) { source @skip(if: TRUE) }
+      }
+    "}
+
+    it "works with field" do
+      id_argument = schema.types['Query'].fields['cheese'].get_argument('id')
```

Comment:
Any suggestions about better alternatives to declare the expected value?

Reply:
This all seems fine to me. If breaks later on, it will break loudly, and it will be straightfoward to update at that time.